proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties

```
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Olet jo yhdistänyt tälle palvelimelle!
+velocity.error.already-connected-proxy=Olet jo yhdistänyt tälle välityspalvelimelle!
+velocity.error.already-connecting=Yrität jo tällä hetkellä yhdistää palvelimeen!
+velocity.error.cant-connect={0} ei saatu yhteyttä: {1}
+velocity.error.connecting-server-error=Yhteyttä palvelimeen {0} ei voitu muodostaa. Yritä myöhemmin uudelleen.
+velocity.error.connected-server-error=Yhteydessäsi palvelimeen {0} tapahtui virhe.
+velocity.error.internal-server-connection-error=Virhe tapahtui sisäisessä palvelinyhteydessä.
+velocity.error.logging-in-too-fast=Kirjaudut sisään liian nopeasti, yritä uudelleen hetken kuluttua.
+velocity.error.online-mode-only=Et ole kirjautuneena Minecraft tilillesi. Jos olet kirjautuneena sisään, yritä pelin uudelleenkäynnistämistä.
+velocity.error.player-connection-error=Yhteydessäsi tapahtui sisäinen virhe.
+velocity.error.modern-forwarding-needs-new-client=Tämä palvelin on yhteensopiva vain version 1.13 ja ylemmän kanssa.
```

"ja" is the wrong word to use here and "ylemmän" sounds unnatural
```suggestion
velocity.error.modern-forwarding-needs-new-client=Tämä palvelin on yhteensopiva vain version 1.13 tai uudemman kanssa.
```